webauthn/registration/verify_registration_response.py

```diff
@@ -91,6 +93,8 @@ def verify_registration_response(
     Raises:
         `helpers.exceptions.InvalidRegistrationResponse` if the response cannot be verified
     """
+    if isinstance(credential, (str, bytes, bytearray)):
+        credential = parse_registration_credential_json(credential)
```

Comment:
I think we should omit `bytes` and `bytearray` from here as well (and the same above) to align with the changes to the `credential` argument type

Reply:
of course. Fixed.